### PR TITLE
Mark read posts in tags owed with a different arrow

### PR DESCRIPTION
**Pull request by Throne3d**
Fix for issue #83. Discussion about this issue appears [here](http://alicorn.elcenia.com/board/viewtopic.php?f=12&t=459&start=1840#p31858) and [here](http://alicorn.elcenia.com/board/viewtopic.php?f=12&t=459&start=1850#p31885), and I think it's an acceptable change.

## Screenshots
The first post (temp4) has been marked unread. The second post (temp3) has some unread replies. The third and fourth posts (temp2 and temp1) have been fully read.

**Starry light**
image

**Starry dark**
image

### CODE REVIEW

File: app/views/posts/_list_item.haml [hunk 1]
```diff
@@ -11,9 +11,14 @@
     - if !current_user.try(:hide_warnings) && post.content_warnings.exists?
       = image_tag "/images/exclamation.png", class: 'vmid', title: "Content Warning: " + post.content_warnings.pluck(:name).join(', '), alt: '!'
   %td.post-subject{class: klass}
-    - if logged_in? && (@show_unread || (!@unread_ids.nil? && @unread_ids.include?(post.id)))
+    - if logged_in? && ((@opened_ids.nil? || !@opened_ids.include?(post.id)) || (!@unread_ids.nil? && @unread_ids.include?(post.id)))
```
Comment (Throne3d) [inline app/views/posts/_list_item.haml]: Oh, oops – I think this line should use `(!@opened_ids.nil? && !@opened_ids.include?(post.id))`

File: app/views/posts/_list_item.haml [hunk 2]
```diff
@@ -11,9 +11,14 @@
     - if !current_user.try(:hide_warnings) && post.content_warnings.exists?
       = image_tag "/images/exclamation.png", class: 'vmid', title: "Content Warning: " + post.content_warnings.pluck(:name).join(', '), alt: '!'
   %td.post-subject{class: klass}
-    - if logged_in? && (@show_unread || (!@unread_ids.nil? && @unread_ids.include?(post.id)))
+    - if logged_in? && ((!@opened_ids.nil? && !@opened_ids.include?(post.id)) || (!@unread_ids.nil? && @unread_ids.include?(post.id)))
```
Comment (Marri) [inline app/views/posts/_list_item.haml]: This is pretty much unreadable but legibility can be fixed later as long as it works

Comment (Marri) [inline app/views/posts/_list_item.haml]: (should probably clarify that these are my personal gripes with things like "line is too long" and "too many conditions" and "lots of parentheses", my existing style set no good precedent here, this is not your fault I'm just griping to gripe)

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -53,6 +53,13 @@ def unread_img
     '/images/bullet_go.png'
   end
 
+  def lastlink_img
```
Comment (Marri) [inline app/helpers/application_helper.rb]: I suspect we'll need a cucumber test for this but I can show you how to make those later

File: app/views/posts/_list_item.haml (hunk 2 above)
Comment (Throne3d) [inline app/views/posts/_list_item.haml]: I agree that it's mostly illegible (which is why I added the comment on the next line). I'm not sure what to do to fix that except to try to parse the bits manually or else somehow extract it into a method.

**Pull request merged**